src/components/editMeasure/populationCriteria/riskAdjustment/qiCore/RiskAdjustment.scss

```diff
@@ -18,6 +18,11 @@
       > label {
         font-family: Rubik, sans-serif;
       }
+      > .risk-description {
+        margin-bottom: 10px;
+        width: 100%;
+        height: 55px;
```

Comment:
does this fixed height not impact the dynamic resize?

Reply:
it doesn't do anything, seemingly 🤷